helm/librechat/values.yaml

```diff
@@ -0,0 +1,215 @@
+# Default values for librechat.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+global:
+  librechat:
+    # existing Secret for all envs/ only Passwords. Can be locally generated with: kubectl create secret generic librechat-secret-envs --from-env-file=.env.example --dry-run=client -o yaml > secret-envs.yaml
+    ## For better maintainabillity, you can put all vars directly in the config Section and only overwrite Secrets with this if nessesary.
+    # Required Values:
+    # - MEILI_MASTER_KEY
+    existingSecretName: "librechat-credentials-env"
+    # Used for Setting the Right Key, can be something like AZURE_API_KEY, if Azure OpenAI is used
+    existingSecretApiKey: OPENAI_API_KEY
+
+image:
+  repository: danny-avila/librechat
+  registry: ghcr.io
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+librechat:
```

Comment:
Generally I am a bit confused why we have global.librechat and librechat?! Feels a bit redundant, can we merge them and have only one? Preferably only librechat on the root level.

Reply:
global is a directive defined by helm. If there are values that should be available everywhere you opt for the global directive. The Secret Names and API Key may also be used in the subchart

Reply:
Could we the librechat stuff from the root level then also to the global.librechat?